lib/WP_Auth0_LoginManager.php

```diff
@@ -356,35 +356,29 @@ private function do_login( $user, $userinfo, $is_new, $id_token, $access_token )
 
     $secure_cookie = is_ssl();
 
-    /**
-     * Filters whether to use a secure sign-on cookie.
-     *
-     * @since 3.1.0
-     *
-     * @param bool  $secure_cookie Whether to use a secure sign-on cookie.
-     * @param array $credentials {
-     *     Array of entered sign-on data.
-     *
-     *     @type string $user_login    Username.
-     *     @type string $user_password Password entered.
-     *     @type bool   $remember      Whether to 'remember' the user. Increases the time
-     *                                 that the cookie will be kept. Default false.
-     * }
-     */
+    // See wp_signon() for documentation on this filter
```

Comment:
Ah that's lazy 😛 I don't see it in this diff. Was this a copy of that method documentation? What's the reason behind removing this one?

Reply:
Not really lazy, more like an unnecessary duplicate of WP core documentation. Seems silly to have that here and in the core location and try to keep the two updated.

Reply:
That's fair. 👍